README.md

```diff
@@ -82,6 +82,17 @@ Use `--update-dependencies` to run `helm dependencies update` *once* before star
 helm-templexer render --update-dependencies my-app.yaml
 ```
 
+### Pipe Manifest Output Through Tool(s) Before Writing to File
+
+Use `--pipe` to pass the manifest output through a tool or set of tools before writing to a file.
+
+```shell
```

Comment:
The examples are misleading because they do not apply to a real use case (the first one) or are also broken (second one).

I propose to start with a single example that is about the usage of `kbld`. (Another good candiate would be `kustomize`).

Currently you used `-f manifest.yaml`, however the file does not exist at the time `kbld` is called. We need to tell the tool to read from `stdin` by using `-f -`. So the example should be written as:

```shell
helm-templexer render --pipe="kbld -f -" my-app.yaml
```

However, the files written to disk now show a problem that is possibly coming from #36. It looks to me like both `stdout` and `stderr` are written to disk:

```yaml
resolve | final: nginx:latest -> index.docker.io/library/nginx@sha256:a05b0cdd4fc1be3b224ba9662ebdf98fe44c09c0c9215b45f84344c12867002e
---
apiVersion: v1
kind: ServiceAccount
metadata:
  labels:
    app.kubernetes.io/instance: my-app
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: nginx-chart
    app.kubernetes.io/version: 1.16.0
    helm.sh/chart: nginx-chart-0.1.0
  name: my-app-nginx-chart
# snip ...
````

Of course, this can be undone by redirecting `stderr`, but I wonder if we cannot improve our code:

```shell
helm-templexer render --pipe="kbld -f - 2> /dev/null" my-app.yaml
```

To verify, I turned this part of `run_helm`:

```rust
let result = Exec::shell(cmd)
  .stdout(Redirection::Pipe)
  .stderr(Redirection::Merge)
  .capture()?;
```

into:

```rust
let result = Exec::shell(cmd)
  .stdout(Redirection::Pipe)
  // .stderr(Redirection::Merge)
  .capture()?;
```

which solves the problem. However we should handle `stderr`.